src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -211,29 +217,68 @@ export class LearningObjectsComponent
    * @memberof LearningObjectsComponent
    */
   async getLearningObjects() {
+    const allStatuses = [
+      'UNRELEASED',
+      'WAITING',
+      'REVIEW',
+      'ACCEPTED_MAJOR',
+      'ACCEPTED_MINOR',
+      'PROOFING',
+      'REJECTED',
+      'RELEASED',
+    ];
+
     if (!this.allResultsReceived) {
       // we know there are more objects to pull
       this.loading = true;
 
-      await this.publicLearningObjectService
-        .getLearningObjects(this.query)
-        .then(val => {
-          this.learningObjects = val.learningObjects;
-
-          if (this.learningObjects.length === val.total) {
-            this.allResultsReceived = true;
-          }
-          this.lastPage = Math.ceil(val.total / 20);
-        })
-        .catch(error => {
-          this.toaster.error(
-            'Error!',
-            'There was an error fetching Learning Objects. Please try again later.'
-          );
-        })
-        .finally(() => {
-          this.loading = false;
-        });
+      if (this.query.username && this.query.username.length > 0) {
+        await this.searchService
+          .getUsersLearningObjects(this.query.username, {
+            ...this.query,
+            // too lazy to lowercase allStatuses values
+            status: allStatuses.map((v) => v.toLowerCase()),
```

Comment:
i didn't wanna use draftsOnly here because then we'd have to add draft LOs onto the list of released LOs and that would break pagination, so I just append all the statuses instead..

there might be a way to append draft LOs with released LOs without breaking pagination i briefly looked at.. i'd be happy to re-visit that solution again if needed

Reply:
Appending the statuses is fine here from what I can tell.

Reply:
If we are doing toLowerCase we should just make the array as lowercase statuses, but also I think there is enum somewhere that has it stored that way already.